src/main/java/edu/hillel/homework/lesson9/warehouse/fruit_storage_area/Lemon.java

```diff
@@ -0,0 +1,11 @@
+package edu.hillel.homework.lesson9.warehouse.fruit_storage_area;
+
+import edu.hillel.homework.lesson9.warehouse.Fruit;
+
+public class Lemon extends Fruit {
+
+    public Lemon() {
+        final float weight = 0.8f;
```

Comment:
One small detail - better to init weight as a static global parameter in the class. It's a constant

Reply:
Yes, I originally thought the same, but at the last moment I decided to insert it into the constructor.
And my decision was influenced first of all by the fact that for some reason the IDE often offers me to initialize such constants in the constructor.
Although I absolutely agree logically, the constant looks prettier when it is displayed at the beginning of the class as a global static field.